### ISSUE TITLE
Loading NLTK corpus like Wordnet from blobs

### ISSUE THREAD

**Issue by priyaananthasankar** [nltk/nltk#2127]
nltk.data.load("URL") works fine if there are specific items to be loaded like a pickle file or stopwords, Wordnet is a compiled library and there needs to be some way to load this from a blob storage.

In AWS Lambda, the set of zipped files can contain the wordnet compiled binaries and can be loaded.
But if wordnet needs to be imported on runtime from NLTK from the cloud, the best way to do it is from a blob, (without using nltk.download() and dealing with system paths)

This issue is raised to support corpora loading from blobs.

**Comment by alvations** [comment]
Good idea to manage data with cloud but we're still figuring how to deal with data management. 

Discussion on https://groups.google.com/d/msg/nltk-dev/LjThWkAthwc/VpVQ56XzBgAJ but it seems like no one else is picking up the discussion =(

BTW, I'm confused. Are you referring Azure Blob or AWS S3?  =)

**Comment by priyaananthasankar** [comment]
Azure Blob. So I wanted stopwords. I just took the english stopwords and manually uploaded in Azure blob and then did nltk.data.load("azure_blob_url"). Well stopwords is an array of words, a txt file so it loaded. I did the same for PunktSentenceTokenizer as well which was pickled.

But I can't do this for wordnet. Either way, it seems hacky :) AWS has a different story on how they load. they want the user to provide a zip which contains all of this.

The point remains that NLTK can take a bunch of arguments to develop a better story on how the corpora can be loaded as some kind of CDN access.
